cmd/rpcdaemon/commands/eth_api.go

```diff
@@ -66,7 +66,7 @@ func (api *APIImpl) Syncing(ctx context.Context) (interface{}, error) {
 		return false, err
 	}
 
-	currentBlock, _, err := stages.GetStageProgress(api.dbReader, stages.TxPool)
+	currentBlock, _, err := stages.GetStageProgress(api.dbReader, stages.Finish)
```

Comment:
oh. didn't know we store this stage progress.

Reply:
We did not do before, but now we do :)